components/login/login_controller/login_controller.jsx

```diff
@@ -372,7 +371,14 @@ class LoginController extends React.Component {
                         style={brandImageStyle}
                     />
                     <div>
-                        {messageHtmlToComponent(formattedText, false, {mentions: false, imagesMetadata: null})}
+                        <Markdown
+                            message={text}
+                            options={
+                                {mentions: false,
+                                    imagesMetadata: null}
+                            }
+                            emojiMap={this.props.emojiMap}
```

Comment:
This can actually be removed since you made the Markdown component get `emojiMap` itself

Reply:
nice catch, this one went right pass me 🤦‍♀